Automattic/jetpack
# Boost: general cleanup and documentation update

#### Changes proposed in this Pull Request:

* Remove Jetpack Boost Docker infrastructure setup and remove the docker exception from the gitattributes file.
* Remove obsolete npm commands, re-organize and standardize npm commands.
* Remove old composer scripts.
* Update documentation.
* Remove obsolete Github actions.

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


N/A

#### Testing instructions:
* There is nothing to test per se. Just make sure the documentation is correct (See the rendered documentation at https://github.com/Automattic/jetpack/tree/cleanup/boost/projects/plugins/boost#readme) and that the Composer Scripts and pnpm commands are working.
